<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="14">
     <widget class="QListView" name="file_view"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="back">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QPushButton" name="init_from_dir">
      <property name="text">
       <string>init from dir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QPushButton" name="add_file">
      <property name="text">
       <string>add file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
